lib/utils/HTTPUtils.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by AsherS on 8/24/15.
+ */
+
+var qfs = require("q-io/fs");
+var networkClient = require('../NetworkClientFactory').getNetworkClient();
+var config = require('../Configuration');
+
+var HTTPUtils = (function(){
+
+    var obj = {};
```

Comment:
you can just write 
return {
downloadFile : function(...) {...}
}
but it's up to you :)